langium/er-dsl/src/language/MetaModel/Relationship.ts

```diff
@@ -43,12 +63,28 @@ export class Relationship extends MultiRelationship {
     }
 }
 
+function getCardinality(side: RelationshipConnection): string {
+    if (side.lower_cardinality == side.upper_cardinality){
+        return `${convertAsteriskToN(side.lower_cardinality)}`;
+    }else {
+        return `(${convertAsteriskToN(side.lower_cardinality)},${convertAsteriskToN(side.upper_cardinality)})`;
+    }
+}
+
+function convertAsteriskToN(cardinality: number | "*"): string | number {
+    if (cardinality == "*"){
+        return "n";
+    }
+    return cardinality;
+}
+
 /**
  * Returns true if the cardinality is singular. If strict is true, it will only return true if the cardinality is exactly 1.
  * When strict is false, it will return true if the cardinality is 0 or 1.
  * @param cardinality The input cardinality to check
  * @param strict If true, the cardinality must be exactly 1 to return true
  */
+
```

Comment:
I dont like this newline. I think it should be attached to the function to be a docstring?